actors/builtin/miner/expiration_queue_test.go

```diff
@@ -61,9 +663,45 @@ func TestExpirationsEmpty(t *testing.T) {
 	require.Equal(t, expected, result)
 }
 
+func testSector(expiration, number, weight, vweight, pledge int64) *SectorOnChainInfo {
```

Comment:
Nit: I this sector number as first arg would be slightly better (but your call to change or not)